app/selectors/view.js

```diff
@@ -1,3 +1,3 @@
 import Immutable from 'immutable';
 
-export const getViewState = (state, viewId) => state.getIn(['dataView', viewId]) || new Immutable.Map();
+export const getViewState = (state, viewId) => Immutable.fromJS(state).getIn(['dataView', viewId]) || new Immutable.Map();
```

Comment:
I suppose − no, but then the question why, why we can't have it the same in all the cases :thinking: